eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/BarData.Serialization.cs

```diff
@@ -18,7 +18,7 @@
 namespace MgmtTypeSpec
 {
     /// <summary> Concrete tracked resource types can be created by aliasing this type using a specific property type. </summary>
-    public partial class BarData : IJsonModel<BarData>
+    public partial class BarData : TrackedResourceData, IJsonModel<BarData>
```

Comment:
this doesn't change the API since the other partial class already implements this